app/src/main/java/com/x8bit/bitwarden/data/platform/util/AuthenticatorBridgeException.kt

```diff
@@ -0,0 +1,6 @@
+package com.x8bit.bitwarden.data.platform.util
+
+/**
+ * Represents a non-fatal authenticator bridge exception.
+ */
+class AuthenticatorBridgeException : Exception()
```

Comment:
So I'm not sure this is even needed. I could just new up some kind an `IllegalStateException` or even just an `Exception` instead. I've done this in the past and I remember it helped on the filtering side in the crashlytics console.

Reply:
🪙 🪙 Passing an `IllegalStateException` with a descriptive message achieves the same thing with less overhead.